src/app/@ansyn/ansyn/modules/overlays/overlay-status/effects/overlay-status.effects.ts

```diff
@@ -174,11 +175,31 @@ export class OverlayStatusEffects {
 					const featurePolygons = polygons.map((polygon) => {
 						return feature(polygon);
 					});
-					const combinedResult = unifyPolygons(featurePolygons);
-					if (combinedResult.geometry.type === 'MultiPolygon') {
-						scannedArea = combinedResult.geometry;
-					} else {	// polygon
-						scannedArea = geojsonPolygonToMultiPolygon(combinedResult.geometry);
+					let combinedResult = unifyPolygons(featurePolygons);
+					let scannedAreaContainsExtentPolygon = false;
+					
+					scannedArea.coordinates.forEach(coordinates => {
+						let multyPolygon = JSON.parse(JSON.stringify(scannedArea));
+						multyPolygon.coordinates = [coordinates];
+						
+						if (getPolygonIntersectionRatioWithMultiPolygon(position.extentPolygon, multyPolygon)) {
+							scannedAreaContainsExtentPolygon = true;
+						}
+					});
+
+					if (scannedAreaContainsExtentPolygon) {
+						combinedResult = difference(combinedResult, position.extentPolygon);
+					}
+					
+					if (combinedResult === null) {
+						scannedArea = null;
+					}
+					else {
+						if (combinedResult.geometry.type === 'MultiPolygon') {
```

Comment:
move this entire else section to the if (scannedAreaContainsExtentPolygon)  above